docs/Installation.md

```diff
@@ -1,8 +1,121 @@
-# Installation of AppStudio E2E Mode in OpenShift CI
+# RHTAP Installation locally and Openshift CI
 
-The following guide will walk through the deployment of AppStudio E2E mode in Openshift CI Pull Requests jobs
+The following guide will walk through the necessary instruction about driving an installation of RHTAP locally or in Openshift CI.
 
-## Install e2e binary in openshift-ci and use pairing PRs feature
+## RHTAP in dev mode
```

Comment:
Why do we need 2 headings which essentially describe the same thing? Can we please keep one?

Reply:
We have level 1 with the title and then we have subtitle with level 2. Is not the same thing